models/experimental/functional_stable_diffusion/README.md

```diff
@@ -0,0 +1,18 @@
+## functional_stable_diffusion Demo
+## How to Run
+
```

Comment:
Please put a clarification stating that the user must build the project, activate the environment, and set the appropriate environment variables.

Reply:
Updated it. 
Thank you